### DISCUSSION TITLE
sublime text failed to uninstall

### DISCUSSION THREAD

**Discussion by aaryamvn** [microsoft/winget-cli#806] [General]
sublime text failed to uninstall

**Comment by suptejas** [comment]
For software that's going to be released in early may, this is worrying!

Yeah I'm facing the same issue:


![image](https://user-images.githubusercontent.com/63039748/111569940-ba609480-87c9-11eb-8621-81737d24faaa.png)

**Reply by ghost** [reply]
Uninstall doesn't work that way,    you have to `winget list`  ,,,, then `winget uninstall "the package/software name (derived from list command) you want to uninstall"`

**Comment by ghost** [comment]
Uninstall doesn't work that way,    you have to `winget list`  ,,,, then `winget uninstall "the package/software name (derived from list command) you want to uninstall"`

**Reply by suptejas** [reply]
Alright that's very very confusing but it works. Thanks.
